package-lock.json

```diff
@@ -41321,10 +41321,10 @@
     },
     "packages/backend": {
       "name": "@clerk/backend",
-      "version": "0.21.0",
```

Comment:
🔧 Please revert the changes in the `package-lock.json` file. We will make the appropriate change.